core/src/main/scala/chisel3/Aggregate.scala

```diff
@@ -1246,16 +1266,6 @@ package experimental {
 
   class BundleLiteralException(message: String) extends ChiselException(message)
   class VecLiteralException(message: String) extends ChiselException(message)
-
-  /** Indicates that the compiler plugin should generate [[cloneType]] for this type
-    *
-    * All user-defined [[Record]]s should mix this trait in as it will be required for upgrading to Chisel 3.6.
-    */
-  trait AutoCloneType { self: Record =>
```

Comment:
are we going to do a deprecating backport of this?

Reply:
I think so, @jackkoenig thoughts?

Reply:
Please do not delete the trait, leave it here but deprecate it, something like
```scala
@deprecated("Auto-clonetype is now always on, you no longer need to mix it in, "Chisel 3.6")
trait AutoCloneType { self: Record =>
}

The Chisel 3.5 work is already done (adding autoclonetype to 3.5 sort of is the "backport")

This trait was only added in 3.5.5 and is step 1 in auto clonetype for Records.
Step 2 is upgrading to 3.6.0 where autoclonetype is always enabled and the trait is deprecated.
Step 3 we'll delete the trait in Chisel 5.